src/Files.App/Helpers/Navigation/NavigationHelpers.cs

```diff
@@ -200,6 +201,10 @@ public static async Task<bool> OpenPath(string path, IShellPage associatedInstan
 					break;
 
 				case FilesystemItemType.File:
+					// Starts the screensaver in full-screen mode
+					if (isScreenSaver)
```

Comment:
Is there a better approach?

Reply:
I guess you could do `args += "/s"` just below, other than that I don't see any improvements

Reply:
Good suggestion